wiki/en/misc/1-index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Jamulus ‒ Play music online. With friends. For free."
+lang: "en"
+permalink: /
+layout: mainhomepage
+ldjAppCategory: "Music"
+ldjSlogan: "Play music online. With friends. For free."
+ldjCreator: "Volker Fischer and contributors"
+metadescription: "Jamulus is free and open source software that lets musicians perform music, rehearse or jam together, in real time over the Internet."
+mAltProgIcon: "Jamulus icon"
+mTSlogan: "Play music online. With friends. For free."
+mTGetStartedNow: "Get started now!"
+mTDownloadNow: 'Download now for'
+mTPlatformsAnd: 'and'
+mTOtherPlatforms: 'other platforms'
+---
+[//]: # Markdown sections should not be indented
```

Comment:
Interesting

Reply:
It gets treated as a code block otherwise.

Reply:
I think it would be good to add the explanation too.

Reply:
Done.